frontend/packages/ceph-storage-plugin/integration-tests-cypress/mocks/deploymentData.ts

```diff
@@ -0,0 +1,37 @@
+export const testDeployment = 'test-deployment';
+
+export const deployment = {
+  apiVersion: 'apps/v1',
+  kind: 'Deployment',
+  metadata: {
+    name: testDeployment,
+  },
+  spec: {
+    selector: {
+      matchLabels: {
+        app: 'hello-openshift',
+      },
+    },
+    replicas: 3,
```

Comment:
```suggestion
    replicas: 1,
```
More replicas take more time to get ready.